packages/intl-datetimeformat/src/types.ts

```diff
@@ -43,7 +43,20 @@ export type RawDateTimeLocaleInternalData = Omit<
   DateTimeFormatLocaleInternalData,
   'formats'
 > & {
-  formats: Record<string, string[]>;
+  formats: Record<
```

Comment:
There is an open issue with the spec about it [not handling different calendars ](https://github.com/tc39/proposal-intl-datetime-style/issues/55). There is an open comment on the [PR to the ecma402](https://github.com/tc39/ecma402/pull/457#discussion_r450232228) to address this. I did take the liberty of assuming what this format will be which could possibly change.

Reply:
I think u should follow the spec as is with the bug intact. It makes it easier to account for the bug fix once the PR is merged upstream

Reply:
I can do this, to make it work it would assume the data is coming from the gregorian calendar regardless of which calendar was passed in. I think this will be fine for now since the following exists in the code 
```
  invariant(
    calendar === 'gregory',
    'We only support Gregory calendar right now'
  );
```